File: 40 - Determine Unique Elements in a Series/solution.py
def unique_elements(arr):
    # Write your code here
    seen = set()
    result = []

    for str in arr:
        if str not in seen:
            seen.add(str)
            result.append(str)

    return result


# Test Cases    
print(unique_elements(["apple", "banana", "apple", "orange", "banana", "kiwi"]))

File: 40 - Determine Unique Elements in a Series/solution.js
function uniqueElements(arr) {
  // Write your code here
  let seen = new Set();
  let result = [];

  for (let str of arr) {
    if (!seen.has(str)) {
      seen.add(str);
      result.push(str);
    }
  }

  return result;
}

// Test Cases
console.log(
  uniqueElements(["apple", "banana", "apple", "orange", "banana", "kiwi"])
);
